cloudwatchmon/cli/put_instance_stats.py

```diff
@@ -167,7 +170,9 @@ def __add_metric_dimensions(self, name, unit, value, common_dims, dims):
             self.names.append(name)
             self.units.append(unit)
             self.values.append(value)
-            self.dimensions.append(dict(common_dims.items() + dim.items()))
+            final_dims = common_dims.copy()
+            final_dims.update(dims)
```

Comment:
This results in a runtime error:
ERROR: dictionary update sequence element #0 has length 1; 2 is required